src/mixins/child-apps.js

```diff
@@ -103,9 +103,10 @@ export default {
    * @method startChildApp
    */
   startChildApp(appName, options) {
-    this.getChildApp(appName).start(options);
+    const childApp = this.getChildApp(appName);
```

Comment:
I don't have any particular allegiance to `const`. I think you've got a good point here. It's readable and concise enough that I'm comfortable with `return this.getChildApp(appName).start(options);`